example/src/Examples/Stickers/Stickers.tsx

```diff
@@ -33,10 +33,12 @@ export const Stickers = () => {
       <GestureHandler
         matrix={helloMatrix}
         dimensions={HelloStickerDimensions}
+        debug
```

Comment:
Same here - move to separate PRs for updates in Examples - these can be merged without any reviews as well.